execution_workflows/APAtrap/bin/postprocessing.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+import pandas as pd
+
+def parse_args(args=None):
+    Description = "Reformat APAtrap identifyDistal3UTR bed file"
+    Epilog = "Example usage: python postprocessing.py <FILE_IN>"
+
+    parser = argparse.ArgumentParser(description=Description, epilog=Epilog)
+    parser.add_argument("FILE_IN", help="Input deAPA output txt file.")
+    return parser.parse_args(args)
+
+
+def reformat_bed(file_in):
+    """
+    This function reformats the txt deAPA output file to files for
+    identification, quantification, and differential challenges
+    :param file_in: txt file to be reformatted
+    :return: N/A
+    """
+    identification_out = open("apatrap_identification_output.bed", "wt")
+    quantification_out = open("apatrap_quantification_output.bed", "wt")
+    differential_out = open("apatrap_differential_output.tsv", "wt")
```

Comment:
I don't think output filenames should be hardcoded. Would be better to define output file name when call script at command line and have option to specify output filename in config file